include/seqan/graph_types/graph_impl_directed.h

```diff
@@ -139,6 +194,53 @@ _getEdgeIdManager(Graph<Directed<TCargo, TSpec> > const& g) {
     return const_cast<TEdgeIdManager&>(g.data_id_managerE);
 }
 
+//////////////////////////////////////////////////////////////////////////////
+
+template<typename TCargo>
+inline void
+_copyGraph(Graph<Directed<TCargo, WithSourceId> > const& source,
+           Graph<Directed<TCargo, WithSourceId> >& dest,
+           bool transpose)
+{
```

Comment:
in SeqAn the argument order is always `(dest1, dest2, src1, src2, option, tag)`, please adapt this even if the function is "private".